swiftchat.go

```diff
@@ -135,20 +95,30 @@ func HandleRoomMessages(w http.ResponseWriter, r *http.Request) {
 			http.StatusBadRequest)
 		return
 	}
-	var jsonBody []byte
+	msgs, err := FormatResp(rooms[id].Messages, w, r)
+	if err != nil {
```

Comment:
err is handled inside format func.. why resp is written outside? 
you should choose one place for it, either write all what you want to Response Writer inside FormatResp or move all write logic out of it. Im thinking outside is preferable, it will allow you to make FormatResp signature easier:
`func FormatResp(payload interface{}, pretty bool) ([]byte, error)`
and then usage:
```
...
_, pretty := r.URL.Query()["pretty"]
resp, err := FormatResp(rooms[id].Messages, pretty)
...
```